test/test_io.py

```diff
@@ -76,23 +76,23 @@ def test_write_read_video(self):
         with temp_video(10, 300, 300, 5, lossless=True) as (f_name, data):
             lv, _, info = io.read_video(f_name)
             assert_equal(data, lv)
-            self.assertEqual(info["video_fps"], 5)
+            assert info["video_fps"] == 5
 
-    @unittest.skipIf(not io._HAS_VIDEO_OPT, "video_reader backend is not chosen")
+    @pytest.mark.skipif(not io._HAS_VIDEO_OPT, reason="video_reader backend is not chosen")
     def test_probe_video_from_file(self):
         with temp_video(10, 300, 300, 5) as (f_name, data):
             video_info = io._probe_video_from_file(f_name)
-            self.assertAlmostEqual(video_info.video_duration, 2, delta=0.1)
-            self.assertAlmostEqual(video_info.video_fps, 5, delta=0.1)
+            assert pytest.approx(2, rel=0.0, abs=0.1) == video_info.video_duration
```

Comment:
If you look at the reference for [`pytest.approx`](https://docs.pytest.org/en/6.2.x/reference.html#pytest-approx), it is always used on the right argument of the comparison. This also aligns better with other check functions for example [`torch.testing.assert_close(actual, expected)`](https://pytorch.org/docs/1.9.0/testing.html?highlight=assert_close#torch.testing.assert_close).

Since this has no functional difference, this is non-blocking.

Reply:
Thanks for the information. I will take that in note.